libtransmission/web.cc

```diff
@@ -777,7 +779,7 @@ class tr_web::Impl
 
     static inline auto curl_init_flag = std::once_flag{};
 
-    std::multimap<uint64_t /*tr_time_msec()*/, CURL*> paused_easy_handles;
+    std::map<CURL*, uint64_t /*tr_time_msec()*/> paused_easy_handles;
```

Comment:
Will it work in the non ordinary case where one is downloading two different torrents with the same weburl, then pauses one torrent but not the other?

Reply:
It should. The curl easy handles are one per task, so for the situation you're describing I think the two torrents would have two different curl easy handles despite both referring to the same URL..